Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -152,11 +155,71 @@ PRIVATE
 )
 
 if(WIN32)
-  target_sources(dolphin-emu PRIVATE DolphinQt.manifest)
+  target_sources(dolphin-emu PRIVATE DolphinQt.manifest DolphinQt.rc)
+
+	if (CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(Dolphin_NAME "DolphinD")
+  else()
+    set(Dolphin_NAME "Dolphin")
+  endif()
+
+  set_target_properties(dolphin-emu PROPERTIES
+    WIN32_EXECUTABLE TRUE
+    OUTPUT_NAME ${Dolphin_NAME}
+  )
+
+  target_compile_options(dolphin-emu PRIVATE /D _SILENCE_CXX17_RESULT_OF_DEPRECATION_WARNING)
+
+  # Copy Sys dir
+  set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS "${CMAKE_SOURCE_DIR}/Data/Sys")
+  file(GLOB_RECURSE resources RELATIVE "${CMAKE_SOURCE_DIR}/Data" "${CMAKE_SOURCE_DIR}/Data/Sys/*")
+
+  foreach(res ${resources})
+    configure_file("${CMAKE_SOURCE_DIR}/Data/${res}" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${res}" COPYONLY)
+  endforeach()
+
+	# Copy qt.conf
+  configure_file(qt.conf.win "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/qt.conf" COPYONLY)
+
+  # Copy plugins
+	set (Qt5_PLUGINS_DIR "${Qt5_DIR}/../../../plugins")
```

Comment:
Indent?
Also: space between `set` and the opening bracket; everything else does not do that.